src/content/docs/import/csv.md

```diff
@@ -13,10 +13,13 @@ end of the the `COPY FROM` clause. The following table shows the configuration p
 |:-----|:-----|:-----|
 | `HEADER` | Whether the first line of the CSV file is the header. Can be true or false. | false |
 | `DELIM` | Character that separates different columns in a lines. | `,`|
+| `LIST_DELIM` | Character that separates different elements in a list, struct, or map. | `,` |
+| `UNBRACED` | Whether list columns are expected to start with `[` and end with `]`. If `true`, braced lists can still be read to allow for nested list scanning | `false` |
```

Comment:
`LIST_DELIM` and `UNBRACED` are also not part of the cherry-picked commits along with `SAMPLE_SIZE`, which should be released later with csv sniffing.